charts/ubiquitous-journey/Chart.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
+- name: argo-cd
+  repository: https://tylerauerbeck.github.io/argo-helm
+  version: 2.0.0
+digest: sha256:d08310eca4c97a8aaad5581641a2fb021810f24d2d9bb616313e1e246a82245e
+generated: "2020-03-24T01:24:24.762740089-04:00"
```

Comment:
@tylerauerbeck  - should this lock file be checked in? Seems auto generated. Im guessing same as Node, it should be as it manages and pins dependencies of dependencies etc